caddy/caddy_setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+export DEBIAN_FRONTEND=noninteractive
+
+# Install Docker
+if [ ! $(which docker) ]; then
+    echo "----Installing docker----"
+    apt update
+    apt install -y apt-transport-https ca-certificates curl software-properties-common
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    add-apt-repository -y "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+    apt update
+    apt install -y docker-ce
+fi
+
+# Run the Docker container
+docker run -d -p 8080:8080 -p 80:9898 -v /tmp/Caddyfile:/etc/caddy/Caddyfile  caddy/caddy:alpine
```

Comment:
Caddy 2 is a beta version. Can you change this to a Caddy 1 install?

```
curl https://getcaddy.com | bash -s personal
caddy -conf /tmp/Caddyfile &
```

Reply:
Right, I was trying to keep the tests even as all the other tests pull from Docker